packages/components/color-picker/src/color-picker.vue

```diff
@@ -1,5 +1,72 @@
 <template>
+  <DefineTemplate>
+    <div
+      :aria-disabled="noTrigger ? colorDisabled : undefined"
```

Comment:
Regarding the parameters or classes configured with the `disabled` attribute, there should be no need to first determine `noTrigger`.
```suggestion
      :aria-disabled="colorDisabled || undefined"
```

Reply:
done